table/block_based/block_based_table_builder.cc

```diff
@@ -1214,53 +1214,9 @@ void BlockBasedTableBuilder::ComputeBlockTrailer(
     const Slice& block_contents, CompressionType compression_type,
     ChecksumType checksum_type, std::array<char, kBlockTrailerSize>* trailer) {
   (*trailer)[0] = compression_type;
-  uint32_t checksum = 0;
```

Comment:
Does this change the include dependency for this function?  Can we simplify where the hash routines are included now?